src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -166,7 +165,7 @@ else if (file.exists())
      * @return the absolute path to the file.
      */
     public static File absolute(File file) {
-        return replacePath(file, absolutePath(file));
+        return new File(absolutePath(file));
```

Comment:
It may be worth checking if we have to do this at all.

Should open a new issue for this.

Reply:
I opened #4543 to track this issue.